runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -108,6 +108,7 @@
 	private static final Pattern DEADMAN_HALF_TELEBLOCK_PATTERN = Pattern.compile("<col=4f006f>A Tele Block spell has been cast on you by (.+). It will expire in 1 minute, 15 seconds.</col>");
 	private static final Pattern FULL_TELEBLOCK_PATTERN = Pattern.compile("<col=4f006f>A Tele Block spell has been cast on you by (.+). It will expire in 5 minutes, 0 seconds.</col>");
 	private static final Pattern HALF_TELEBLOCK_PATTERN = Pattern.compile("<col=4f006f>A Tele Block spell has been cast on you by (.+). It will expire in 2 minutes, 30 seconds.</col>");
+	private static final Pattern KILLED_TELEBLOCK_OPPONENT = Pattern.compile("<col=4f006f>Your Tele Block has been removed because you killed (.+).</col>");
```

Comment:
We don't actually end up using the matched group at all, so this would be much better off being a `String`, and matched using `event.getMessage().startsWith(KILLED_TELEBLOCK_OPPONENT_TEXT)`:

```suggestion
	private static final String KILLED_TELEBLOCK_OPPONENT_TEXT = "<col=4f006f>Your Tele Block has been removed because you killed ";
```